docs/components/concepts/clusters.md

```diff
@@ -25,16 +26,16 @@ When you deploy and execute your [BPMN](/components/modeler/bpmn/bpmn.md) or [DM
 
 Free Trial clusters have the same functionality as a production cluster, but are size small and only available during your trial period. You cannot convert a Free Trial cluster to a different kind of cluster.
 
-Once you sign up for a Free Trial, you are able to create one production cluster for the time of your trial.
+Once you sign up for a Free Trial, you are able to create one production cluster for the duration of your trial.
 
-When your Free Trial plan expires, you are automatically transferred to the Free Plan. This plan allows you to model BPMN and DMN collaboratively, but does not support execution of your models. Any cluster created during your free trial is deleted, and you cannot create new clusters.
+When your Free Trial plan expires, you are automatically transferred to the Free Plan. This plan allows you to model BPMN and DMN collaboratively, but does not support execution of your models. Any cluster created during your trial is deleted, and you cannot create new clusters.
 
 ## Development clusters
 
 Development clusters are recommended for development, testing, proof of concepts, and demos.
 
-Starter Plan users have one **development cluster**, with free execution for development, included in their plan.
-Deployment and execution of models (process instances, decision instances, and task users) is provided at no cost.
+Starter Plan users have one **development cluster**, with free execution for development included in their plan. Deployment and execution of models (process instances, decision instances, and task users) is provided at no cost.
+
 Additional clusters can be purchased through your [billing reservations](/components/console/manage-plan/update-billing-reservations.md).
 
 Additionally, the following applies to **development clusters**:
```

Comment:
Do we need to update this auto-pause note below, or is it still relevant?

Reply:
@felix-mueller are development clusters part of your PM scope? Can you confirm this note is still relevant?

Reply:
@jprcoliveira sounds like this is in your scope! 😄 Is this auto-pause note still relevant? 

```
:::caution
**Cluster auto-pause** is not yet available. Development clusters will be paused if they go unused for two hours. When a cluster is paused, not all functionality will work, including BPMN timers and BPMN message catch events.
:::
```

Reply:
@akeller this note is still relevant. Hopefully by 8.4/8.5 we have this in place already